modules/ROOT/pages/deploy-maven-4.x.adoc

```diff
@@ -3,7 +3,7 @@
 Runtime Fabric supports using Maven to manage and deploy Mule applications using Maven. The information in this topic is specific to deploying an application to Mule runtime, version 4.
```

Comment:
Anypoint Runtime Fabric supports using Apache Maven...Maven: specifically, applications to Mule runtime engine version 4.

QUESTION: first sentence says Mule apps using Maven; second sentence says apps TO Mule. Are these Mule apps, or apps to Mule, or are these the same thing? Should we clarify?

Reply:
I have clarified - thank you!